.github/workflows/release.yml

```diff
@@ -74,29 +73,35 @@ jobs:
         run: pip install -r requirements.txt
 
       - name: Install pyinstaller
-        run: pip install pyinstaller
+        run: pip install pyinstaller==5.0.1
```

Comment:
Should the version used in `build.sh` be updated as well? It seems like having only a single version of pyinstaller to consider would make the maintenance of the system more simple. That is of course unless there is a specific technical reason why 4.2 must be used in `build.sh`

https://github.com/arduino/imgtool-packing/blob/f5cbf1bdfccc5963684aedc659de879f9283097f/build.sh#L8

Reply:
Ups, I completely forgot about this